### PR TITLE
On branch walker-branch

### PR DESCRIPTION
**Pull request by walker9296**
Changes to be committed:
	new file:   contracts/bubbleSort.scrypt

Bubble sort
1) source file
     bubbleSort.scrypt
2) Function
     static function sort(Int[ARRAY_ITEM_MAX] arr) : Int[ARRAY_ITEM_MAX]
        arr the int array to be sorted
        rtn sorted int array
3) Use limitations, the definition of the length of the array
     static const int ARRAY_ITEM_MAX = 10;

### CODE REVIEW

File: contracts/bubbleSort.scrypt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                        arr[j] = temp;
+                    }
+                }
+            }
+        }
+        return arr;
+    }
+}
+
+contract bubbleSortTest {
```
Comment (zhfnjust) [inline contracts/bubbleSort.scrypt]: Modify to start with a capital letter

File: contracts/bubbleSort.scrypt [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
+library bubbleSort {
```
Comment (zhfnjust) [inline contracts/bubbleSort.scrypt]: Modify to start with a capital letter

File: contracts/bubbleSort.scrypt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        
+        require(arr2[0] == -200);
+        require(arr2[9] == 100);
+    }
+
+    public function test2() {
+        int[bubbleSort.ARRAY_ITEM_MAX] arr1 = [1, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+        int[bubbleSort.ARRAY_ITEM_MAX] arr2 = bubbleSort.sort(arr1);
+        
+        require(arr2[0] == 0);
```
Comment (zhfnjust) [inline contracts/bubbleSort.scrypt:40]: It's better to check the whole array：

```
require(arr2 == [0, 0, 0, 0, 0, 0, 0, 0, 0, 1]);
```

File: contracts/bubbleSort.scrypt [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        return arr;
+    }
+}
+
+contract bubbleSortTest {
+    public function test1() {
+        int[bubbleSort.ARRAY_ITEM_MAX] arr1 = [-200, 0, 100, -1, 3, 60, 3, 45, 11, -10];
+        int[bubbleSort.ARRAY_ITEM_MAX] arr2 = bubbleSort.sort(arr1);
+        
+        require(arr2[0] == -200);
```
Comment (zhfnjust) [inline contracts/bubbleSort.scrypt:27]: It's better to check the whole array：

```
require(arr2 == [-200, ...]);
```

**Comment by walker9296** [comment]
Insertion sort
1. source file
    insertionSort.scrypt
2. Function
    static function sort(Int[ARRAY_ITEM_MAX] arr) : Int[ARRAY_ITEM_MAX]
        arr:          int[] to be sorted
        return value: sorted int[]
3. Use limitations, the definition of the length of the array
    static const int ARRAY_ITEM_MAX = 5;

File: contracts/insertionSort.scrypt [hunk 5]
```diff
@@ -0,0 +1,44 @@
+
+library insertionSort {
```
Comment (zhfnjust) [inline contracts/insertionSort.scrypt]: Modify to start with a capital letter

File: contracts/insertionSort.scrypt [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                        }
+                    }
+                }
+            }
+        }
+        return arr;
+    }
+}
+
+contract insertionSortTest {
```
Comment (zhfnjust) [inline contracts/insertionSort.scrypt]: Modify to start with a capital letter

File: contracts/insertionSort.scrypt [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    public function test() {
+        int[insertionSort.ARRAY_ITEM_MAX] arr1 = [60, 3, 45, 11, -10];
+        int[insertionSort.ARRAY_ITEM_MAX] arr2 = insertionSort.sort(arr1);
+        require(arr2[0] == -10);
+        require(arr2[4] == 60);
+
+        arr1 = [1, 0, -1, 0, 0];
+        arr2 = insertionSort.sort(arr1);
+        require(arr2[0] == -1);
```
Comment (zhfnjust) [inline contracts/insertionSort.scrypt:39]: like above

File: contracts/insertionSort.scrypt [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+        require(arr2[4] == 1);
+
+        arr1 = [-200, 0, 100, -1, 300];
+        arr2 = insertionSort.sort(arr1);
+        require(arr2[0] == -200);
+        require(arr2[4] == 300);
```
Comment (zhfnjust) [inline contracts/insertionSort.scrypt:51]: like above

**Review by zhfnjust — COMMENTED** [review]
good ,let's do a small fix

**Pull request merged**